### DISCUSSION TITLE
我做了一个乞丐版的BiBiGPT——“太长不看”

### DISCUSSION THREAD

**Discussion by raines1220** [JimmyLv/BibiGPT#127] [Show and tell]
## 太长不看版
我做了一个免费版的BiBiGPT——太长不看，网站有类似的功能，但是总体来说比较简陋。
因此，**不建议在日常工作或学习中使用太长不看**，但是如果想试毒可以点这里：[https://www.taichangbukan.cn](https://www.taichangbukan.cn)

## 废话不说，先上网站的功能截图：
<img width="733" alt="taichangbukan" src="https://github.com/JimmyLv/BibiGPT/assets/31393931/4e6681c7-7b57-48ce-b170-577dedee2932">

## 然后是浏览器插件的截图：
<img width="1050" alt="image" src="https://github.com/JimmyLv/BibiGPT/assets/31393931/1a64f8c1-4ceb-4233-9f3b-b15e6f53aba2">

## 与BiBiGPT 有什么不同
* **目前免费**：我自己正好有个2500刀的闲置的初创企业号，搞这个好像也没有什么成本，就先免费开放吧? 
* **分章节总结**: 相比于常见的bullet points的总结方式，太长不看是按内容的“章节”分别总结的，整体数据结构大概是这样的: 
```
{
            "摘要": "XXXX",
            "章节": [
                      "章节1":   {
                                 "标题": "阿巴阿巴",
                                 "细节" : "哈哈哈哈哈哈"
                                 "开始": 0
                                 "结束": 10
                      },
                     .......
            ]
      }
```
* **浏览器插件**: 太长不看除了网站以外，还有一个浏览器的插件，可以在B站一边看视频一边总结，然后跳转时间什么的倒还算方便。不过可能并不实用，原因下面就会说。

## “乞丐版”目前有什么局限性？
* **仅支持B站的视频总结**，不支持自己上传视频，因为感觉不知道怎么实现，感觉有点麻烦。
* **不支持自定义总结的粒度、格式**，AI跑出来啥样就是啥样了，主要就图个省credit。
* **不支持和视频聊天**，写完现在的功能已然筋疲力尽，没有力气了再写别的功能了。
* **不支持流式的输出，总结非常慢。** 如果总结一个之前没有见过的视频，因为内部极其臃肿、低效的代码，目前最少可能需要3分钟。建议普通人遇到新视频没有见过就直接弃了，否则可能会因为极长的等待时间，导致身心受到极大的伤害。
* **目前不开源。** 倒不是敝帚自珍，主要是这个项目写的太赶了，代码一团糟，心中难免有种羞耻感。日后有时间重构一下代码，然后应该就可以开源了。
* **有每日限额。** 目前每个B站帐号，登录之后可以总结50个视频。未来根据情况，我可能会降低这个额度。没有办法，生活窘迫，我也没有太多的办法~

## 想要联系我
* 如果对网站有什么想吐槽的，或者什么建议，可以加我微信：raines1220
* 如果想付费使用……目前不支持。假话是我很喜欢用爱发电，真话是这个功能我不会写，学了一下觉得有点烧脑，所以放弃了~

**Comment by JimmyLv** [comment]
哇，为你打Call！！
